admin_guide/sdn_troubleshooting.adoc

```diff
@@ -495,6 +495,54 @@ normal operation, so do not be surprised to see entries there.
 
 Check external firewalls, if any, allow traffic to the target address (this is site-dependent, and beyond the purview of this guide).
 
+== Debugging Virtual Networking
```

Comment:
Add manual/unique IDs for all headings.

If there's a worthwhile way we can have some intro matter here so that it doesn't go from one heading straight to a subheading, that might be cool.